src/assets/styles/_variables.scss

```
@@ -1 +1,8 @@
 $primary-color: #6372F1;
+$p-light-grey: rgba(0,0,0,0.62);
+$floralwhite:  #FFFAF0;
+$black: rgba(0, 0, 0, 0.12);
+$black-icon: rgba(0, 0, 0, 0.64);
+$american-violet: #551b8c;
```

Please give all in caps
```suggestion
$american-violet: #551B8C;
```

---

resolved...................Thank you